server/builds.go

```diff
@@ -207,7 +206,7 @@ func (b *Builds) checkBuildLink(ctx context.Context, s *Server, pr *model.PullRe
 		select {
 		case <-ctx.Done():
 			s.sendGitHubComment(pr.RepoOwner, pr.RepoName, pr.Number, "Timed out waiting for build link. Please check the logs.")
-			return "", fmt.Errorf("timed out waiting the build link")
+			return "", errors.Errorf("timed out waiting the build link")
```

Comment:
Minor nit pick: `Errorf` is not needed here, `New` is enough
```suggestion
			return "", errors.New("timed out waiting the build link")
```